x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -151,114 +92,83 @@ export function ServiceOverviewTransactionsTable(props: Props) {
-        'xpack.apm.serviceOverview.transactionsTableColumnName',
-        {
-          defaultMessage: 'Name',
-        }
-      ),
-      render: (_, { name, transactionType: type }) => {
-        return (
-          <TruncateWithTooltip
-            text={name}
-            content={
-              <TransactionDetailLink
-                serviceName={serviceName}
-                transactionName={name}
-                transactionType={type}
-                latencyAggregationType={latencyAggregationType}
-              >
-                {name}
-              </TransactionDetailLink>
-            }
-          />
-        );
-      },
-    },
-    {
-      field: 'latency',
-      name: getLatencyAggregationTypeLabel(latencyAggregationType),
-      width: px(unit * 10),
-      render: (_, { latency }) => {
-        return (
-          <SparkPlot
-            color="euiColorVis1"
-            compact
-            series={latency.timeseries ?? undefined}
-            valueLabel={asMillisecondDuration(latency.value)}
-          />
-        );
-      },
-    },
-    {
-      field: 'throughput',
-      name: i18n.translate(
-        'xpack.apm.serviceOverview.transactionsTableColumnThroughput',
-        { defaultMessage: 'Throughput' }
-      ),
-      width: px(unit * 10),
-      render: (_, { throughput }) => {
-        return (
-          <SparkPlot
-            color="euiColorVis0"
-            compact
-            series={throughput.timeseries ?? undefined}
-            valueLabel={asTransactionRate(throughput.value)}
-          />
-        );
-      },
-    },
-    {
-      field: 'errorRate',
-      name: i18n.translate(
-        'xpack.apm.serviceOverview.transactionsTableColumnErrorRate',
-        {
-          defaultMessage: 'Error rate',
-        }
-      ),
-      width: px(unit * 8),
-      render: (_, { errorRate }) => {
-        return (
-          <SparkPlot
-            color="euiColorVis7"
-            compact
-            series={errorRate.timeseries ?? undefined}
-            valueLabel={asPercent(errorRate.value, 1)}
-          />
-        );
-      },
+  const transactionNames = JSON.stringify(
+    currentPageTransactionGroups.map(({ name }) => name).sort()
+  );
+
+  const {
+    data: transactionGroupComparisonStatistics,
+    status: transactionGroupComparisonStatisticsStatus,
+  } = useFetcher(
+    (callApmApi) => {
+      if (
+        currentPageTransactionGroups.length &&
+        start &&
+        end &&
+        transactionType
+      ) {
+        return callApmApi({
+          endpoint:
+            'GET /api/apm/services/{serviceName}/transactions/groups/comparison_statistics',
+          params: {
+            path: { serviceName },
+            query: {
+              start,
+              end,
+              uiFilters: JSON.stringify(uiFilters),
+              numBuckets: 20,
+              transactionType,
+              latencyAggregationType: latencyAggregationType as LatencyAggregationType,
```

Comment:
We should avoid forcing the ts compiler like this if possible.

```suggestion
              latencyAggregationType: latencyAggregationType as LatencyAggregationType,
```

In this case `latencyAggregationType` is optional but we manually override this introducing the risk if a runtime bug.

Can you change the type of `latencyAggregationType` to `LatencyAggregationType` instead of string in x-pack/plugins/apm/public/components/shared/Links/url_helpers.ts ?

It's not perfect since we also don't validate that. But at least we'll be able to catch null/undefined errors.
With the new client we'll hopefully get better runtime validations